MolkPackageQueue/Program.cs

```diff
@@ -2,15 +2,46 @@
 {
     internal class Program
     {
+        public static int numberOfOrderProcessed = 0;
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Implement MPS");
-            // Instantiate the MPS-PriorityQueue
-            // Create a function to queue and dequeue packages according to the rules. 
-            // Don´t forget the logging lists
-            // Print log for packages created in order of creation, with payload packageName and package priority
-            // Print log for packages handled (dequeue and add to logg), same content as above.
-            // No high prio should be in bottom of handled list, alla paket som skapas ska finnas i hanterad-listan.
+            Random rnd = new Random();
+            List<Package> inComming = new List<Package>();
+            RecivePackage recivePackage = new RecivePackage();
+            PriorityQueue priorityQueue = new PriorityQueue();
+
+            // Creates a list of random number of packages with randomized priority and Payload
+            inComming = recivePackage.ReciveIncomming();
+            Console.WriteLine();
+
+            // Sorts the incoming list by priority
+            var sortedPackages = inComming.OrderByDescending(p => p.Priority).ToList();
```

Comment:
Sorting before enqueue:ing is not necessary. The queue:s will be sorted correctly internally if every package is added to correct queue from start.